x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/helpers.ts

```diff
@@ -20,3 +20,16 @@ export const validateAlertTagsArrays = (tags: AlertTags, ids: string[]) => {
   }
   return validationErrors;
 };
+
+export const validateAlertAssigneesArrays = (assignees: AlertAssignees, ids: string[]) => {
+  const validationErrors = [];
+  if (ids.length === 0) {
+    validationErrors.push(i18n.NO_IDS_VALIDATION_ERROR);
```

Comment:
I think we can have this validation in the type itself by specifying that it should be a non-empty string array.

Reply:
done